__tests__/app.test.js

```diff
@@ -260,3 +260,29 @@ describe("DELETE /api/comments/:comment_id,", () => {
       });
   });
 });
+
+describe("GET /api/users", () => {
+  test("200: returns all the users with the desired categories", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
```

Comment:
Whenever we are expecting an array we always want to have an assertion to ensure it is not an empty array as well, so we dont get a false passing test when given an empty array